examples/lib/app.js

```diff
@@ -1,5 +1,7 @@
 var vumigo = require('vumigo_v02');
-var LocationState = require('../../lib');
+var location = require('../../lib');
+var LocationState = location.LocationState;
+var GoogleMaps = location.GoogleMaps;
```

Comment:
Was the intention to add a seperate example app using OpenStreetMaps in another issue?